classes/help.php

```diff
@@ -42,18 +41,18 @@ function print_page() {
 								<a name="premium_features"></a>
 								<div class="postbox">
 
-									<h3><?php _e( 'Premium Features', 'SimpleMap' ); // XSS ok. ?></h3>
+									<h3><?php _e( 'Premium Features', 'simplemap' ); // XSS ok. ?></h3>
 
 									<div class="inside" style="padding: 0 10px 10px 10px;">
 
 										<div class="table">
 											<table class="form-table">
 
 												<tr>
-													<td><?php _e( 'Custom category markers can now be used. Login to Premium support and look for the KB article for instructions.', 'SimpleMap' ); // XSS ok. ?></td>
+													<td><?php _e( 'Custom category markers can now be used. Login to Premium support and look for the KB article for instructions.', 'simplemap' ); // XSS ok. ?></td>
 												</tr>
 												<tr>
-													<td><?php _e( 'We now have a search widget. Login to Premium support and look for the KB article for instructions.', 'SimpleMap' ); // XSS ok. ?></td>
+													<td><?php _e( 'We now have a search widget. Login to Premium support and look for the KB article for instructions.', 'simplemap' ); // XSS ok. ?></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 202 characters

<!-- Comment by [SideCI](https://sideci.com) -->